examples/dem/small-scale-rotating-drum-postprocessing/example_small_rotating_drum.py

```diff
@@ -0,0 +1,53 @@
+#Post-processing tool to track mixing in small rotating drum
+
+# import sys
+import sys
+
+#Path to the module
+path_to_module = '../../../contrib/postprocessing/'
+sys.path.append(path_to_module)
+
+#Import tools from module
+from lethe_pyvista_tools import *
+
+
+#Create object named "particles"
+particles = lethe_pyvista_tools(".", "small-rotating-drum-dem.prm")
+
+#Read files and store data to object named "particles"
+particles.read_lethe_to_pyvista("out.pvd")
+
+
+#State condition for particle_color array creation
+condition = "(y**2 + z**2)**(1/2) > 0.025"
+
+#Create array named particle_color and give the value of 1 to all particles within the given condition
+#The packing of particles is finished after 40th time step
```

Comment:
Is it really the 40th time step? Is it not the dt*frequency of output? Or the 40th output file. 
Maybe "reference_time_step" is a bit unclear. What do you think?